File: step_by_step/step_5/2577.py
# 숫자의 개수

A = int(input())
B = int(input())
C = int(input())

product = A*B*C
digits = list(str(product))
# print(digits) : 예제의 경우, ['1', '7', '0', '3', '7', '3', '0', '0']
for x in range(0,10): # x 는 0~9
    print(digits.count(str(x))) # digits는 list[str]이므로 str(x)를 count

File: step_by_step/step_5/c/2577.c
#include <stdio.h>

int main(void){
    int A, B, C;
    scanf("%d %d %d", &A, &B, &C);
    
    int product = A*B*C;
    int digits[10] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
    while(product>0){
        int digit = product%10;
        digits[digit] = digits[digit] + 1;
        product = product / 10;
    }

    for(int i=0; i<10; i++){
        printf("%d\n", digits[i]);
    }
}